index.js

```diff
@@ -43,7 +43,7 @@ module.exports = function serialize(obj) {
         }
 
         return value;
-    });
+    }, space);
```

Comment:
This isn't right, the replacer is the second arg, `space` is the third: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify